dali/test/python/operator/test_random_resized_crop.py

```diff
@@ -145,7 +147,8 @@ def _test_rrc(device, max_frames, layout, aspect_ratio_range, area_range, output
         input = input.gpu()
     out = fn.random_resized_crop(input, random_aspect_ratio=aspect_ratio_range,
                                  random_area=area_range, size=output_size,
-                                 interp_type=dali.types.INTERP_LINEAR, seed=12321,
+                                 interp_type=dali.types.INTERP_LINEAR,
+                                 antialias=False, seed=12321,
```

Comment:
this was not necessary but it silents a warning and the default behavior for INTERP_LINEAR is now different, so this restores the no-antialiasing behavior as when this testcase was introduced